### PR TITLE
Getting used with gloang

### PR DESCRIPTION
**Pull request by flamarion**
This is my first time with Golang after some introductory training and a lot of Stackoverflow "research".
I want to get a review before moving forward to avoid too many items to fix.

### CODE REVIEW

File: pkg/deployments/terraform/privatecloud/general.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	config.Region = region
+}
+
+func EnableRedisConfig(config *GeneralConfig) {
+	enable, _ := pterm.DefaultInteractiveConfirm.Show("Would you like to enable external Redis?")
+	if !enable {
+		return
+	}
+
+	config.EnableRedis = enable
```
Comment (jsbroks) [inline pkg/deployments/terraform/privatecloud/general.go:71]:
```suggestion
	config.EnableRedis = enable
```

File: pkg/deployments/terraform/privatecloud/general.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+func UseInternalQueue(config *GeneralConfig) {
+	enable, _ := pterm.DefaultInteractiveConfirm.Show("Would you like to enable external queue (SNS)?")
+	if !enable {
+		return
+	}
+
+	config.EnableRedis = enable
```
Comment (jsbroks) [inline pkg/deployments/terraform/privatecloud/general.go:80]:
```suggestion
	enable, _ := pterm.DefaultInteractiveConfirm.Show("Would you like to enable external queue (SNS)?")
	config.EnableRedis = enable
```

File: pkg/deployments/utils.go [hunk 3]
```diff
@@ -18,8 +19,13 @@ func GetTerraformTemplate(name string) string {
 
 	defer resp.Body.Close()
 
-	b, err := io.ReadAll(resp.Body)
+	b, _ := io.ReadAll(resp.Body)
 	pterm.Fatal.PrintOnError()
```
Comment (jsbroks) [inline pkg/deployments/utils.go:23]:
```suggestion
	b, err := io.ReadAll(resp.Body)
	pterm.Fatal.PrintOnError(err )
```

File: templates/cloud_aws.tf [hunk 4]
```diff
@@ -8,7 +8,7 @@ locals {
   database_engine_version = "{{.Database.Version}}"
   database_instance_class = "{{.Database.Size}}"
 
-  deletion_protection = {{.DeletionProtection}}
```
Comment (jsbroks) [inline templates/cloud_aws.tf:11]: You need this without quotations

**Comment by flamarion** [comment]
Is it ok to create a `General` section in the `yaml` like the example below? 

```
    production:
        engine: Terraform
        platform: Amazon Web Services
        private-cloud:
            general:
                namespace: dad
                region: us-east-1
                enableredis: true
                useinternalqueue: false
```

If it's ok, I will move all general configurations to there.

**Comment by venky-wandb** [comment]
Looks good to me! 👍

**Pull request merged**